README.md

```diff
@@ -30,35 +30,63 @@
+Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 LTS). Windows and macOS are also supported, but you might experience more issues.
+
+### 1. Download OpenAndroidInstaller
+
+> [!WARNING]
+> ARM-based systems are **unsupported**.
+
+Follow the instructions for your desktop OS. The executables are compressed inside `.zip` files, so you'll have to extract them first (make sure you have extracting software installed). If you get prompted to, you'll have to adjust the permission of the executable to ensure its proper functionality. 
+
+#### Linux
+
+You can download OAI from either:
+
+1. From Flathub: [`org.openandroidinstaller.OpenAndroidInstaller`](https://flathub.org/apps/org.openandroidinstaller.OpenAndroidInstaller) (recommended)
+2. GitHub Releases: [`openandroidinstaller-ubuntu-latest.zip`](https://github.com/openandroidinstaller-dev/openandroidinstaller/releases/download/latest/openandroidinstaller-ubuntu-latest.zip)
+
+#### Windows
+
+You can download OAI for Windows from the GitHub Releases: [`openandroidinstaller-windows.zip`](https://github.com/openandroidinstaller-dev/openandroidinstaller/releases/download/latest/openandroidinstaller-windows.zip)
+
+You might also need to [install the Universal USB Drivers](https://adb.clockworkmod.com) and other potentially drivers needed for your device.
+
+#### macOS
+
+For macOS, you can get OpenAndroidInstaller from the GitHub Releases: [`openandroidinstaller-macos.zip`](https://github.com/openandroidinstaller-dev/openandroidinstaller/releases/download/latest/openandroidinstaller-macos.zip)
+
+### 2. Download the custom ROM image & the TWRP recovery image (& optionally addons)
+
+Here are the official links for:
+
+  - some custom ROMs:
+    - [LineageOS](https://wiki.lineageos.org/devices)
+    - [/e/OS](https://doc.e.foundation/devices)
+    - [LineageOS for MicroG](https://download.lineage.microg.org)
+    - [BlissRoms](https://blissroms.org)
+    - [PixelExperience](https://download.pixelexperience.org)
+  - TWRP Recovery:
+    - [TWRP recovery](https://twrp.me/Devices)
+  - Optional Addons:
+    - There are different packages of *Google Apps* available:
+      - [MindTheGapps](https://wiki.lineageos.org/gapps#downloads)
+      - [NikGApps](https://nikgapps.com)
+    - [MicroG](https://microg.org)
+      - The recommended way to install MicroG is from the `.zip` file provided here: [`MinMicroG-abuse-CI/releases`](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
+    - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
+  - or you can just search the web or the [XDA Developers forum](https://xdaforums.com) to find an appropriate version for your device.
+
+### 3. Start the desktop app
+
+There, you will be prompted to plug your device to your computer and you will be given a step-by-step tutorial on how to install the ROM, TWRP and the addons for your specific model.
```

Comment:
Just a rephrasing including a grammar correction:
`After starting the app you will be prompted to plug your device into your computer and you will be given a step-by-step tutorial on how to install your new OS for your specific device.`